src/model.ts

```diff
@@ -80,6 +80,13 @@ export class GitExtension implements IGitExtension {
     }
   }
 
+  /**
+   * Requests the new GET endpoint.
+   */
+  get isIgnored(): Git.IBranch {
+    return;
+  }
```

Comment:
Then to call the backend with the proper argument, you will need something like that

```suggestion
  async isIgnored(pathToCheck: string): Promise<{files: string[]}> {
    const path = await this._getPathRepository();
    return requestAPI<{files: string[]}>(
      URLExt.join(path, 'ignore') +  URLExt.objectToQueryString({path: pathToCheck}),
      'GET'
    );
  }
```